saddle-core/src/main/scala/org/saddle/Vec.scala

```diff
@@ -504,6 +504,20 @@ trait Vec[@spec(Boolean, Int, Long, Double) T] extends NumericOps[Vec[T]] {
     */
   def fillNA(f: Int => T): Vec[T]
 
+  /** Fill NA values by propagating defined values forward.
+    *
+    * @param limit 
+    *   If > 0, propagate over a maximum of `limit` consecutive NA values
+    */
+  def ffill(limit: Int = 0): Vec[T]
```

Comment:
What do you think of adding aliases like fillForward and fillBackward? (Or rename?)